bookr/src/components/PrivateRoute.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+
+const PrivateRoute = ({ component: Component, ...rest }) => (
+  <Route
+    {...rest}
+    render={props =>
+      localStorage.getItem("token") ? (
```

Comment:
I'm not 100% sure, but the auth might end up getting set up a little bit differently if it turns out to be using OAuth. Some slight changes may be needed, but for right now this looks great!